specification.markdown

```diff
@@ -223,6 +223,7 @@ Within an EBML Schema the `<element>` uses the following attributes to define an
 | attribute name | required | definition |
 |:---------------|:---------|:-----------|
 | name           | Yes      | The official human-readable name of the EBML Element. The value of the name MUST be in the form of an NCName as defined by the [XML Schema specification](http://www.w3.org/TR/1999/REC-xml-names-19990114/#ns-decl). |
+| parent         | Yes      | The path to the parent EBML Element. This path MUST be defined with the full hierarchy of EBML Elements separated with a `/`. The top EBML Element in the path hierarchy being the first in the value. Elements that can be found anywhere inside en EBML Element and its children MUST use the `*` (wildcard) value after the path of the first parent allowed. |
```

Comment:
OK, I can live with the parent just above. It requires each name throughout the whole DocType(s) to be unique.

The parent="Tag|SimpleTag" is certainly better clearer, should we remove the recursive flag then ? I suppose the Global attribute covers the case of `*` but what parent should these parents have then ? Or do we also remove the `global` attribute and assume no `parent` equals can be found anywhere ?